src/main/java/me/gnat008/perworldinventory/groups/GroupManager.java

```diff
@@ -162,6 +188,10 @@ public void saveGroupsToDisk() {
             groupsConfigFile.set(groupKey + ".worlds", group.getWorlds());
             // Saving gamemode regardless of management; might be saving after convert
             groupsConfigFile.set(groupKey + ".default-gamemode", group.getGameMode().name());
+            groupsConfigFile.set(groupKey + ".enforceLastWorld", group.shouldUseLastWorld());
+            groupsConfigFile.set(groupKey + ".enforceLastPosInGroup", group.shouldUseLastPosInGroup());
+            groupsConfigFile.set(groupKey + ".enforceLastPosInWorld", group.shouldUseLastPosInWorld());
+            groupsConfigFile.set(groupKey + ".enforceLastPosInWorld", group.getDefaultWorld());
```

Comment:
Oh, totally... This was a last minute change when I noticed my naming convention in the YML file did not match.